test/plugin/test_out_google_cloud.rb

```diff
@@ -316,4 +316,20 @@ def get_number(field)
   def null_value
     nil
   end
+
+  # 'responseSize' and 'requestSize' are Integers in the gRPC proto, yet Strings
+  # in REST API client.
```

Comment:
Want to reference the issue you've opened (https://github.com/google/google-api-ruby-client/issues/619) in a comment here and before `source_location_message`?

Reply:
Sure thing. Just added this:

```
  # TODO(lingshi): Address this accordingly once the following  question is
  # answered: https://github.com/google/google-api-ruby-client/issues/619.
  # If this discrepency is legit, add some comments to explain the reason.
  # Otherwise once the discrepency is fixed, we need to upgrade to that version
  # and change our tests accordingly.
```

Reply:
Changed `lingshi` to `qingling128`.....